python/ct/client/reporter.py

```diff
@@ -104,11 +104,10 @@ class CertificateReport(object):
     __metaclass__ = abc.ABCMeta
 
     def __init__(self, checks=all_checks.ALL_CHECKS,
-                 pool_size=FLAGS.reporter_workers,
```

Comment:
Can you confirm there is no code that creates a Reporter class and passes in these parameters that would be broken by this change?